frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -62,10 +62,18 @@ interface MultiselectOption {
 class Multiselect extends React.PureComponent<Props, State> {
   private readonly formClearHelper = new FormClearHelper()
 
+  maxSelections: number = this.props.element.maxSelections
+
   public state: State = {
     value: this.initialValue,
   }
 
+  public overMaxSelections(): boolean {
+    return this.maxSelections === 0
+      ? false
+      : this.initialValue.length >= this.maxSelections
+  }
```

Comment:
nit: This would be a bit cleaner without the ternary operator as

```javascript
return this.maxSelections > 0 && this.initialValue.length >= this.maxSelections
```